### PR TITLE
blog(add article): tekton + tracetest

### PR DESCRIPTION
**Pull request by adnanrahic**
# Changes

This PR adds a new blog post for using distributed tracing for testing with Tracetest and automating tests with Tekton.
Based on [this docs page in Tracetest docs](https://docs.tracetest.io/ci-cd-automation/tekton-pipeline). 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by adnanrahic** [comment]
Thanks for responding @afrittoli !

I've set the date of the blog post to August 15th. LMK if we need to change that to something else. 😄

**Comment by adnanrahic** [comment]
Hey @AlanGreene  @geriom ! Wanted to touch base and ask if tomorrow is a good date to publish the blog post. If not, let me know and we can work out another date. 😄

File: content/en/blog/2023/distributed-traces-for-testing-with-tekton-pipelines-and-tracetest/index.md [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+
+kubectl apply --filename \
+https://storage.googleapis.com/tekton-releases/dashboard/latest/release.yaml
+```
+
+## 2. Install Tracetest CLI
+
+Install Tracetest CLI by following [these instructions](https://docs.tracetest.io/getting-started/installation) for your OS.
+
+```bash
```
Comment (geriom) [inline content/en/blog/2023/distributed-traces-for-testing-with-tekton-pipelines-and-tracetest/index.md:48]: Provide instructions for all every OS. Something like this should work:

```go-html-template
{{< tabs >}}

{{< tab header="MacOS" lang="bash" >}}
brew install kubeshop/tracetest/tracetest
{{< /tab >}}


{{< tab header="Linux" lang="bash" >}}
curl -L https://raw.githubusercontent.com/kubeshop/tracetest/main/install-cli.sh | bash
{{< /tab >}}


{{< tab header="Windows" lang="bash" >}}
choco source add --name=kubeshop_repo --source=https://chocolatey.kubeshop.io/chocolatey ; choco install tracetest
{{< /tab >}}

{{< /tabs >}}
```

**Review by geriom — CHANGES_REQUESTED** [review]
Sorry fro the delay. Added a couple of suggestions.

**Review by geriom — APPROVED** [review]

**Comment by adnanrahic** [comment]
Hey @geriom 👋 
Thanks for approving the PR!
Wanted to ask when this could be merged? I am not authorized to merge it myself.

**Comment by geriom** [comment]
> Hey @geriom 👋 Thanks for approving the PR! Wanted to ask when this could be merged? I am not authorized to merge it myself.

I just pinged Alan on slack in case he has some time to take a look. PRs usually need two approvals to be merged.

**Comment by adnanrahic** [comment]
Hey @geriom & @AlanGreene ! 👋 

I wanted to circle back to this and ping you to ask if we should update the blog post date and get it merged this week.

Alternatively, I'm happy to wait for @afrittoli to get back from PTO.

Thanks! 🙌

**Review by afrittoli — COMMENTED** [review]
Thank you!
/lgtm

**Pull request merged**